### ISSUE TITLE
Enable Quick Type on Mac/Safari "double tap" Issue

### ISSUE THREAD

**Issue by jmadlun** [strongbox-password-safe/Strongbox#560]
Hi Mark, strange behavior in Mac/Safari when I have the Enable Quick Type checked in Database Settings, Autofill.  I am pretty sure it started with the most recent app update.  In Safari, on any web page, I have to tap twice (two quick taps on the trackpad) to click on or open a link on the page; usually a single tap is all I need to open a link.   If I uncheck Enable Quick Type Safari works as expected.  A single tap opens a link.

Thanks for all your great work on this app, I am a fan.

App Version: com.markmcguill.strongbox.mac [1.16.4 (3041)]
--------------------
Device
--------------------
Model: MacBookAir10,1
CPU: ARM64E
System Name: MacOS
System Version: Version 12.0.1 (Build 21A559)
[autofill_mac.txt](https://github.com/strongbox-password-safe/Strongbox/files/7575614/autofill_mac.txt)

**Comment by strongbox-mark** [comment]
Hi @jmadlun - That sounds very odd. Does this survive restarting Safari or your device? 

Can't think of how something like this could happen. 

Thanks!

**Comment by jmadlun** [comment]
Thanks @strongbox-mark, I did some more trouble shooting last night and it seems to have been an issue with another password autofill browser extension/app.  It was not active, but still installed.  I uninstalled the app and then turned back on the autofill for Strongbox and it works fine now.  So I am thinking it was some kind of autofill conflict or something (not sure??).  Anyway, don't waste any cycles on this one.  I can close it.
